File: python3/412. Fizz Buzz/fizz-buzz.py
class Solution:
    def fizzBuzz(self, n):
      new_list = []
      for x in range(1, n+1):
        if x%3 == 0 == x%5:
          new_list.append("FizzBuzz")
        elif x%3 == 0:
          new_list.append("Fizz")
        elif x%5 == 0:
          new_list.append("Buzz")
        else:
          new_list.append(str(x))
      return new_list
        

File: PHP/412. Fizz Buzz/fizz-buzz.php
class Solution {

    function fizzBuzz($n) {
        $new_list = [];
      for($x =1; $x < $n+1; $x++) {
        
        if($x%3 == 0 && $x%5 == 0) {
          array_push($new_list, 'FizzBuzz');
          }
        elseif($x%3 == 0) {
          array_push($new_list, 'Fizz');
          }
        elseif($x%5 == 0) {
          array_push($new_list, 'Buzz');
          }
        else{
          array_push($new_list, strval($x));
          }
      
      }
      return $new_list;
    
    }
}
